lua/nvim-tree.lua

```diff
@@ -188,6 +188,7 @@ function M.find_file(with_open)
   local bufname = vim.fn.bufname()
   local bufnr = api.nvim_get_current_buf()
   local filepath = vim.fn.fnamemodify(bufname, ':p')
+  filepath = utils.canonical_path(filepath)
```

Comment:
could you avoid reassign ? 
```suggestion
  local filepath = utils.canonical_path(vim.fn.fnamemodify(bufname, ':p'))
```